source/agent/addons/quic/QuicTransportServer.cc

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "QuicTransportServer.h"
+#include "QuicFactory.h"
+#include "QuicTransportStream.h"
+#include "owt/quic/quic_transport_factory.h"
+#include "owt/quic/quic_transport_session_interface.h"
+
+using v8::Function;
+using v8::FunctionTemplate;
+using v8::Local;
+using v8::Object;
+using v8::ObjectTemplate;
+using v8::Value;
+
+Nan::Persistent<v8::Function> QuicTransportServer::s_constructor;
+
+DEFINE_LOGGER(QuicTransportServer, "QuicTransportServer");
+
+QuicTransportServer::QuicTransportServer(int port, const std::string& pfxPath, const std::string& password)
+    : m_quicServer(QuicFactory::getQuicTransportFactory()->CreateQuicTransportServer(port, pfxPath.c_str(), password.c_str()))
```

Comment:
validate the pfxPath before passing to CreateQuicTransportServer.

Reply:
It's validated in JavaScript code here https://github.com/open-webrtc-toolkit/owt-server/blob/5f96db912d6f97b6434d82c7ce3ea109e8e57af6/source/agent/quic/index.js#L91-L96